Applications/PlazaDetails/Reporting.qmd

```diff
@@ -7,24 +7,24 @@ aliases:
 
 This documentation section demonstrates how to use the reporting function within the openEO algorithm plaza. Individuals using the platform can track the services they have used and see how credits have been deducted for each service. The purpose of this guide is to help users efficiently track and document their service consumption while also keeping an eye on their credit balance.
 
-Please check that you are logged in to your account for accessing the reporting functionality. Once logged in, select the "Reporting" option from the navbar, that will take you to the screen shown in the figure below:
+Please ensure that the account is logged in to access the reporting functionality. Once logged in, navigate to the "Reporting" option in the navigation bar. This will take users to the screen displayed in the figure below:
 
 ![](../_images/Reporting.png)
 
-If you haven't executed anything yet, there may not be much information displayed on the screen. However, as you start utilizing openEO and the services on the plaza, the screen will populate with relevant data that can be visualized in the following manner:
+If executions have yet to be performed, the screen may display little information. However, as openEO and plaza services are used, the screen will populate with relevant data that can be visualized as follows:
```

Comment:
Please revise this section. The wording is unclear. Also executions is a poor choice of words for obvious reasons